packages/opentelemetry-core/src/utils/environment.ts

```diff
@@ -16,37 +16,109 @@
 
 import { LogLevel } from '../common/types';
 
-export type ENVIRONMENT_MAP = { [key: string]: string | number };
+const DEFAULT_LIST_SEPARATOR = ',';
 
 /**
  * Environment interface to define all names
  */
-export interface ENVIRONMENT {
-  OTEL_LOG_LEVEL?: LogLevel;
-  OTEL_NO_PATCH_MODULES?: string;
+
+type ENVIRONMENT_NUMBERS = {
   OTEL_SAMPLING_PROBABILITY?: number;
   OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT?: number;
   OTEL_SPAN_EVENT_COUNT_LIMIT?: number;
   OTEL_SPAN_LINK_COUNT_LIMIT?: number;
-}
+};
 
-const ENVIRONMENT_NUMBERS: Partial<keyof ENVIRONMENT>[] = [
+const ENVIRONMENT_NUMBERS_KEYS: Partial<keyof ENVIRONMENT_NUMBERS>[] = [
   'OTEL_SAMPLING_PROBABILITY',
   'OTEL_SPAN_ATTRIBUTE_COUNT_LIMIT',
   'OTEL_SPAN_EVENT_COUNT_LIMIT',
   'OTEL_SPAN_LINK_COUNT_LIMIT',
 ];
 
+function isEnvVarANumber(key: unknown): key is keyof ENVIRONMENT_NUMBERS {
+  return ENVIRONMENT_NUMBERS_KEYS.includes(key as keyof ENVIRONMENT_NUMBERS);
```

Comment:
so README says "Automated browser tests are run in the latest version of Headless Chrome.
There is currently no list of officially supported browsers, but OpenTelemetry is developed using standard web technologies with wide support and should work in currently supported versions of major browsers.", what's missing from Array.proto.includes that you think is a major browser. Node supports this since v6 and README says v8 is lowest supported by OTel.

Reply:
or wait do you mean Array.proto.includes is not standard? But isn't https://tc39.es/ecma262/2020/#sec-array.prototype.includes published already?

Reply:
internet explorer